pkg/BUILD.bazel

```diff
@@ -1,6 +1,7 @@
+# GENERATED FILE DO NOT EDIT
```

Comment:
We can tell git that this file is generated, but adding the following to .gitattributes:
```
BUILD.bazel -diff
```

Reply:
huh, we don't mark other BUILD files are -diff, so why this one?

Reply:
`pkg/BUILD.bazel` is 100% generated (by `generate-test-suites`, i.e., the thing this PR created). It's the ONLY `BUILD.bazel` file in tree that is 100% generated. The other `BUILD.bazel` files are generated, but can be manually updated under certain circumstances as well, so the contents of those diffs matter to anyone reviewing a patch.